deploy/templates/nstemplatetiers/base/cluster.yaml

```diff
@@ -164,6 +164,22 @@ objects:
     name: ${USERNAME}-stage
   spec:
     timeoutSeconds: ${{IDLER_TIMEOUT_SECONDS}}
+- apiVersion: dbaas.redhat.com/v1alpha1
+  kind: DBaaSTenant
+  metadata:
+    annotations:
+      toolchain.dev.openshift.com/optional-resource: "yes"
```

Comment:
```suggestion
      toolchain.dev.openshift.com/optional-resource: true
```

Reply:
I had exactly this at the very beginning, but the annotation wasn't properly processed, because it didn't contain a string but a boolean. 
So, I could have used:
```
toolchain.dev.openshift.com/optional-resource: "true"
```
or even
```
toolchain.dev.openshift.com/optional-resource: 'true'
```
but I wasn't sure if the apostrophes wouldn't be removed, so I chose using `"yes"`